source/clear-linux/get-started/virtual-machine-install/vmware-esxi-preconfigured-cl-image.rst

```diff
@@ -0,0 +1,245 @@
+
+      .. code-block:: console
+
+        # vmkfstools -i clear-[version number]-vmware.vmdk -d zeroedthick clear-[version number]-esxi.vmdk
+
+    Two files should result from this:
+
+    * :file:`clear-[version number]-esxi-flat.vmdk`
+    * :file:`clear-[version number]-esxi.vmdk`
+
+    The :file:`clear-[version number]-esxi.vmdk` file will be used in the next
+    section when you create a new VM.
+
+Create and configure a new VM
+*****************************
+
+In this section, you will create a new VM, configure its basic parameters such
+number of CPUs, memory size, and attach the converted |CL| VMware image.  
+Also, in order to boot |CL|, you must enable UEFI support. 
+
+#.  Under the :guilabel:`Navigator` window, select :guilabel:`Virtual Machines`.
+    See Figure 4.
+#.  On the right window, click the :guilabel:`Create / Register VM` button.
+
+    .. figure:: figures/vmware-esxi/vmware-esxi-preconfigured-cl-image-4.png
+      :scale: 100 %
+      :alt: VMware ESXi - Navigator > Virtual Machines
+
+      Figure 4: VMware ESXi - Navigator > Virtual Machines
+
+#.  On the :guilabel:`Select creation type` step:
+
+    #.  Select the :guilabel:`Create a new virtual machine` option. See Figure 5.
+    #.  Click the :guilabel:`Next` button.
+
+        .. figure:: figures/vmware-esxi/vmware-esxi-preconfigured-cl-image-5.png
+          :scale: 100 %
+          :alt: VMware ESXi - Create a new virtual machine
+
+          Figure 5: VMware ESXi - Create a new virtual machine
+
+#.  On the :guilabel:`Select a name and guest OS` step:
+
+    #.  Give the new VM a name in the :guilabel:`Name` field. See Figure 6.
+    #.  Set the :guilabel:`Compatability` option to :guilabel:`ESXi 6.5 virtual machine`.
+    #.  Set the :guilabel:`Guest OS family` option to :guilabel:`Linux`.
+    #.  Set the :guilabel:`Guest OS version` option to :guilabel:`Other 3.x or later Linux (64-bit)`.
+    #.  Click the :guilabel:`Next` button.
+
+        .. figure:: figures/vmware-esxi/vmware-esxi-preconfigured-cl-image-6.png
+          :scale: 100 %
+          :alt: VMware ESXi - Give a name and select guest OS type
+
+          Figure 6: VMware ESXi - Give a name and select guest OS type
+
+#.  On the :guilabel:`Select storage` step:
+
+    #.  Accept the default option.
+    #.  Click the :guilabel:`Next` button.
+
+#.  On the :guilabel:`Customize settings` step:
+
+    #.  Click the :guilabel:`Virtual Hardware` button. See Figure 7.
+    #.  Expand the :guilabel:`CPU` setting and enable :guilabel:`Hardware virtualization` by
+        checking :guilabel:`Expose hardware assisted virtualization to the guest OS`.
+
+        .. figure:: figures/vmware-esxi/vmware-esxi-preconfigured-cl-image-7.png
+          :scale: 100 %
+          :alt: VMware ESXi - Enable hardware virtualization
+
+          Figure 7: VMware ESXi - Enable hardware virtualization
+
+    #.  Remove the default :guilabel:`Hard drive 1` setting by the clicking
```

Comment:
delete "the": "Remove the default :guilabel:`Hard drive 1` setting by the clicking" -> "Remove the default :guilabel:`Hard drive 1` setting by clicking"